aio/deploy/helm-chart/kubernetes-dashboard/templates/clusterrolebinding-metrics.yaml

```
@@ -0,0 +1,30 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{ if .Values.rbac.clusterRoleMetrics -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: "{{ template "kubernetes-dashboard.fullname" . }}"
+  labels:
+{{ include "kubernetes-dashboard.labels" . | indent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ template "kubernetes-dashboard.fullname" . }}
+subjects:
+  - kind: ServiceAccount
+    name: kubernetes-dashboard
```

In other places used `ServiceAccount`(e.g. [here1](https://github.com/kubernetes/dashboard/pull/4502/files#diff-ed120a3d919cabbd8ade3ae964f3a424R51), [here2](https://github.com/kubernetes/dashboard/pull/4502/files#diff-457ad80f5b951b1fd5851a488d2b7e90R28), and so on), `name` is set to `{{ template "kubernetes-dashboard.serviceAccountName" . }}`. Isn't it needed here also?

---

```suggestion
    name: {{ template "kubernetes-dashboard.serviceAccountName" . }}
```

I can confirm, it is definitely should be like this, otherwise metrics scrapping would not work with following sympthoms:
```
$ kubectl -n monitoring logs dashboard-metrics-scraper-5c7db799bf-2hcrf
192.168.99.248 - - [20/Mar/2020:11:12:12 +0000] "GET / HTTP/1.1" 200 6 "" "kube-probe/1.14+"
{"level":"error","msg":"Error scraping node metrics: nodes.metrics.k8s.io is forbidden: User \"system:serviceaccount:monitoring:dashboard-kubernetes-dashboard\" cannot list resource \"nodes\" in API group \"metrics.k8s.io\" at the cluster scope","time":"2020-03-20T11:12:21Z"}
```

and 

```
$ kubectl -n monitoring logs dashboard-kubernetes-dashboard-649dffc9dc-6vgxh
2020/03/20 10:53:39 Skipping metric because of error: Metric label not set.
```

---

I've re-worked how metrics-scraper behaves and fixed it. Thanks!